editors/code/src/debug.ts

```diff
@@ -203,9 +221,11 @@ async function discoverSourceFileMap(
         const commitHash = rx.exec(data)?.[1];
         if (commitHash) {
             const rustlib = path.normalize(sysroot + "/lib/rustlib/src/rust");
-            sourceFileMap[`/rustc/${commitHash}/`] = rustlib;
+            return { source: rustlib, destination: rustlib };
         }
     }
+
+    return;
```

Comment:
Is this `return` needed?

Reply:
afraid so, yeah. Without this return, we'd get a "Not all code paths return a value. `ts(7030)`" error.